tests/stp/stripe/process.csv

```diff
@@ -1,8 +1,8 @@
 host,port,proctype,procname,U,localtime,g,T,w,load,startwithall,extras,qcmd
 localhost,{KDBBASEPORT}+100,discovery,discovery1,${TORQHOME}/appconfig/passwords/accesslist.txt,1,0,,,${KDBCODE}/processes/discovery.q,1,,q
-localhost,{KDBBASEPORT}+101,segmentedtickerplant,stp1,${TORQHOME}/appconfig/passwords/accesslist.txt,1,0,,,${KDBCODE}/processes/segmentedtickerplant.q,1,-schemafile ${TORQHOME}/database.q,q
+localhost,{KDBBASEPORT}+101,segmentedtickerplant,stp1,${TORQHOME}/appconfig/passwords/accesslist.txt,1,0,,,${KDBCODE}/processes/segmentedtickerplant.q,1,-schemafile ${TORQHOME}/database.q -.ds.numseg 4,q
 localhost,{KDBBASEPORT}+101,segmentedtickerplant,stpperf,${TORQHOME}/appconfig/passwords/accesslist.txt,1,0,,,${KDBCODE}/processes/segmentedtickerplant.q ${perfpath}/prep_test.q,0,-schemafile ${TORQHOME}/database.q,q
-localhost,{KDBBASEPORT}+102,rdb,rdb1,${TORQHOME}/appconfig/passwords/accesslist.txt,1,1,180,,${KDBCODE}/processes/rdb.q,1,-.rdb.replaylog 0b -.rdb.subfiltered 1,q
-localhost,{KDBBASEPORT}+103,rdb,rdb2,${TORQHOME}/appconfig/passwords/accesslist.txt,1,1,180,,${KDBCODE}/processes/rdb.q,1,-.rdb.replaylog 0b -.rdb.subfiltered 1,q
-localhost,{KDBBASEPORT}+104,rdb,rdb3,${TORQHOME}/appconfig/passwords/accesslist.txt,1,1,180,,${KDBCODE}/processes/rdb.q,1,-.rdb.replaylog 0b -.rdb.subfiltered 1,q
-localhost,{KDBBASEPORT}+105,rdb,rdb4,${TORQHOME}/appconfig/passwords/accesslist.txt,1,1,180,,${KDBCODE}/processes/rdb.q,1,-.rdb.replaylog 0b -.rdb.subfiltered 1,q
\ No newline at end of file
+localhost,{KDBBASEPORT}+102,rdb,rdb1,${TORQHOME}/appconfig/passwords/accesslist.txt,1,1,180,,${KDBCODE}/processes/rdb.q,1,-.rdb.replaylog 0b -.rdb.subfiltered 1 -.ds.numseg 4,q
```

Comment:
I don't fully understand this start line - why do the RDBs need -.ds.numseg 4? 
Would they not just need to know which segment they are subscribing to? i.e. the first one needs to subscribe to segment 0, next to segment 1 etc.

Reply:
This is for testing purposes only. In the normal setup it is not required.
In the test, .rdb.replaylog is disabled, so during the testing when a rdb restarts, it manually replays the log and invoke the .rdb.applyfilters[;.rdb.subscribesyms] which requires a .ds.numseg variable (as .rdb.subscribesyms uses .ds.stripe to apply the filter) on the rdb.

Specifically in this line of code:
[run,0,0,q,"(rdbHandles rdbDown) @\: "".rdb.applyfilters[;.rdb.subscribesyms]each .rdb.subtables""",1,,"Apply filters to the restarted RDB(s) process"](https://github.com/AquaQAnalytics/TorQ/blob/385f8628541b9d096d98322c8f6c1cd82266aa76/tests/stp/stripe/stripe.csv#L39)